src/mcedit2/editorcommands/analyze.py

```diff
@@ -0,0 +1,67 @@
+"""
+    analyze
+"""
+from __future__ import absolute_import
+from PySide import QtGui, QtCore
+import logging
+
+from mcedit2.util.load_ui import load_ui
+
+log = logging.getLogger(__name__)
+
+class AnalyzeOutputDialog(QtGui.QDialog):
+    def __init__(self, editorSession, blockCount, entityCount, tileEntityCount, *args, **kwargs):
+        super(AnalyzeOutputDialog, self).__init__(*args, **kwargs)
+        self.editorSession = editorSession
+        self.blocktypes = editorSession.worldEditor.blocktypes
+        
+        load_ui("analyze.ui", baseinstance=self)
+        blockTable = self.blockOutputTable
+        self.setupBlockTable(blockCount, blockTable)
+        
+        entityTable = self.entityOutputTable
+        self.setupEntityTable(entityCount, tileEntityCount, entityTable)
+
+        self.sizeHint()
+        self.exec_()
+
+
+
+    def setupBlockTable(self, blockCount, table):
+        blockCounts = sorted([(self.editorSession.worldEditor.blocktypes[ i & 0xfff, i >> 12], blockCount[i])
+                         for i in blockCount.nonzero()[0]])
+        table.setRowCount(len(blockCounts))
+        table.setColumnCount(4)
+        table.setHorizontalHeaderLabels(['Name', 'ID', 'Data', 'Count'])
+
+        for n, output in enumerate(blockCounts):
+            nameItem = QtGui.QTableWidgetItem(output[0].displayName)
```

Comment:
You will definitely want to use a QTableView with a QAbstractListModel subclass for these tables.